scripts/example-pipelines-test.sh

```diff
@@ -1,3 +1,157 @@
 #!/bin/bash
 
-echo "Temporary test file"
+cp -L $KUBECONFIG /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig
+export IMG=${CI_OPERATOR_IMG}
+
+winImageDownloadURL="http://http-server/disk.img"
+
+# Deploy resources
+echo "Deploying resources"
+./scripts/deploy-resources.sh
+
+# SECRET
+key="/tmp/secrets/accessKeyId"
+token="/tmp/secrets/secretKey"
+
+if test -f "$key" && test -f "$token"; then
+  id=$(cat $key | tr -d '\n' | base64)
+  token=$(cat $token | tr -d '\n' | base64 | tr -d ' \n')
+
+  oc apply -n kubevirt -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tekton-operator-container-disk-puller
+  namespace: kubevirt
+type: Opaque
+data:
+  accessKeyId: "${id}"
+  secretKey: "${token}"
+EOF
+fi
+
+function wait_until_exists() {
+  timeout 10m bash <<- EOF
+  until oc get $1; do
+    sleep 5
+  done
+EOF
+}
+
+function wait_for_pipelinerun() {
+  oc wait -n kubevirt --for=condition=Succeeded=True pipelinerun -l pipelinerun=$1-run --timeout=60m &
+  success_pid=$!
+
+  oc wait -n kubevirt --for=condition=Succeeded=False pipelinerun -l pipelinerun=$1-run --timeout=60m && exit 1 &
+  failure_pid=$!
+
+  wait -n $success_pid $failure_pid
+
+  if (( $? == 0 )); then
+    echo "Pipelinerun $1 succeeded"
+  else
+    echo "Pipelinerun $1 failed"
+    exit 1
+  fi
+}
+
+echo "Creating datavolume with win10 iso"
+oc apply -f "scripts/test-files/win10-dv.yaml"
+echo "Waiting for datavolume to be created"
+oc wait -n kubevirt datavolume iso-dv --timeout=10m --for=jsonpath='{.status.phase}'='Succeeded'
```

Comment:
This will probably now fail with CDI's garbage collection enabled. Can you check if the PVC's imported pod annotation has the value Succeeded instead?

Example:
https://github.com/kubevirt/ssp-operator/blob/60ded49c34220f724f81f8d989958e077d5cd038/tests/misc_test.go#L347